sdk/ai/azure-ai-projects/samples/evaluations/sample_continuous_eval_rule.py

```diff
@@ -0,0 +1,109 @@
+# ------------------------------------
```

Comment:
how about use "evaluation" instead of "eval" in file name, to match other file names?